CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -0,0 +1,424 @@
+`infrastructure` stanza to Gateway objects that is intended to carry
+infrastructure configuration specific to that Gateway object.
+
+[GEP-1762](https://gateway-api.sigs.k8s.io/geps/gep-1762/) adds a section
+for `labels` and `annotations` to this stanza that specifies labels and
+annotations to be annotated to all Kubernetes resources created to fulfill
+that Gateway request.
+
+This feature can be used to affect the labels and annotations created on
+Loadbalancer Services by in-cluster implementations to fulfill the Gateway
+contract.
+
+This feature has graduated to Standard and is now considered GA or Stable.
+
+This feature's name for conformance tests and GatewayClass status reporting
+is `GatewayInfrastructurePropagation`.
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support. Please see the implementation's
+`gatewayclass.status.supportedFeatures` stanza or the 
+[implementation feature list](https://gateway-api.sigs.k8s.io/implementations/v1.2/)
+once the final release is out to see if your implementation supports it.
+
+Relevant PRs:
+* Remove experimental from gateway infrastructure by @keithmattix in [#3272](https://github.com/kubernetes-sigs/gateway-api/pull/3272)
+* Validation for label keys and values according to Kubernetes specification by @snorwin in [#3284](https://github.com/kubernetes-sigs/gateway-api/pull/3284)
+
+### HTTPRoute Timeouts and Durations moved to Standard 🎉
+
+The HTTPRoute `Timeouts` field on Route Rules has graduated to Standard and is
+now considered GA or Stable.
+
+This field allows you to configure overall Request Timeouts as well as Backend
+Request Timeouts. For more information, refer to
+[GEP 1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/).
+
+The relevant feature names this for conformance tests and GatewayClass status
+reporting are:
+* `HTTPRouteRequestTimeout` for `httproute.spec.rules[].timeouts.request`
+* `HTTPRouteBackendTimeout` for `httproute.spec.rules[].timeouts.backendRequest`
+
+This feature's status is **Extended**, meaning that it is _optional_ for
+implementations to support. Please see the implementation's
+`gatewayclass.status.supportedFeatures` stanza or the 
+[implementation feature list](https://gateway-api.sigs.k8s.io/implementations/v1.2/)
+once the final release is out to see if your implementation supports it.
+
+Relevant PRs:
+* Graduate GEP-2257 and GEP-1742 to standard by @kflynn in [#3210](https://github.com/kubernetes-sigs/gateway-api/pull/3210)
+* GEP2257 Duration parsing by @xtineskim in [#3271](https://github.com/kubernetes-sigs/gateway-api/pull/3271)
+
+### BackendProtocol Support moved to Standard 🎉
+
+The previous coordinated work across both Gateway API and upstream
+Kubernetes which defined 3 new values for the AppProtocol field on
+Service Ports has graduated to Standard and is now considered GA or Stable.
+
+The values are:
+* `kubernetes.io/h2c` - HTTP/2 over cleartext as described in
```

Comment:
Nit: A lot of our docs gen requires an empty line before lists. It may not be necessary for changelogs, but just in case:
```suggestion
The values are:

* `kubernetes.io/h2c` - HTTP/2 over cleartext as described in
```